lib/http.js

```diff
@@ -73,14 +73,14 @@ SwaggerHttp.prototype.execute = function (obj, opts) {
 
   var responseInterceptor = function(data) {
     if(opts && opts.responseInterceptor) {
-      data = opts.responseInterceptor.apply(data);
+      data = opts.responseInterceptor.apply(data, [obj]);
```

Comment:
Sorry I didn't reply to this - apply's second arg is an array which is "spread" to the function, so essentially the function gets one argument which is obj, not an array.